demos/unpkg/app/directory.ts

```diff
@@ -1,4 +1,4 @@
-import { detectMimeType } from '@remix-run/mime'
+import { detectMimeType } from 'remix'
```

Comment:
This kind of import makes me think we're exporting too much from the `remix` index file. We should get this from `remix/mime`.